src/app/core/entity-components/entity-list/entity-list.component.ts

```diff
@@ -179,8 +179,8 @@ export class EntityListComponent<T extends Entity>
     }
 
     this.listName =
-      this.listConfig.title ??
-      this.listName ??
+      this.listConfig.title ||
```

Comment:
Any reason why you prefer || over ?? here? From my understanding, I would see ?? as the more logical choice here https://stackoverflow.com/a/61481631/1473411

Reply:
Yes, but all our list titles are empty because the component has a default of `listName = ""` which is then always used instead of the actual plural title of the entity.